content/en/docs/architecture/ci-operator-internals/steps.md

```diff
@@ -1,42 +1,11 @@
 ---
-title: "CI Operator Internals"
-description: A detailed view of some aspects of the ci-operator implementation, intended for developers working on its source code.
+title: "Steps"
+description: A description of the various `ci-operator` task types.
```

Comment:
Do we use "task types" and "step types" interchangeably?

Reply:
That's the term used originally in the [introduction](https://docs.ci.openshift.org/docs/architecture/ci-operator-steps#execution-graph).  The intent there is to describe the system in more general terms that hopefully are better known by someone new to the architecture before using the `ci-operator`-specific terminology.

The text in the package list and the [following section](https://docs.ci.openshift.org/docs/architecture/ci-operator-steps/#step-graph) make the connection between the two terms, and "step" is used exclusively from then on.